apps/web/src/server/applications/pages/case/documentation/applications-documentation.controller.js

```diff
@@ -16,3 +16,15 @@ export async function viewApplicationsCaseDocumentationCategories(request, respo
 
 	response.render(`applications/case/project-documentation`, properties);
 }
+
+/**
+ * View the documentation upload page
+ *
+ * @type {import('@pins/express').RenderHandler<{currentFolder: string}, {}>}
+ */
+export async function viewApplicationsCaseDocumentationUpload(request, response) {
+	// TODO: connect to the :folderId/parents-folder endpoint
```

Comment:
@DavidJWhiteDev, this is where I need the breadcrumbs information. Most likely I will use what you're doing, so could be a good idea to put it in the locals (as like as the case pages do with the case data).